src/hotspots/HotspotMaterial.js

```diff
@@ -266,16 +292,117 @@ FORGE.HotspotMaterial.prototype._createTextureFromImage = function(image)
 {
     this._displayObject = image;
 
-    this.setTextureFrame(image.frame);
+    this._texture = new THREE.Texture();
+    this._texture.generateMipmaps = false;
+    this._texture.minFilter = THREE.LinearFilter;
```

Comment:
We may have a problem here, as hotspots are smoothed again on desktop, but are now really aliased and ugly when in WebVR (seen on GearVR). Either we change this depending on the type of view (normal or vr), either we find a compromise between the two.